### DISCUSSION TITLE
Nova Api documentation?

### DISCUSSION THREAD

**Discussion by emjayess** [laravel/nova-issues#3294] [Q&A] [answered]
Does there exist any hosted Api documentation for Nova? Something more complete than the (woefully incomplete) standard documentation?...

At the moment, I am specifically trying to hunt down details regarding `fill`, `fillForUpdate` methods on a nova resource.

**Comment by davidhemphill** [comment, marked as answer]
Nova’s API isn’t documented or intended for use and is subject to change.

What specifically do you find woefully missing from the existing documentation? That would give us a concrete idea on what to add. Thanks!

**Reply by emjayess** [reply]
> What specifically do you find woefully missing from the existing documentation?

... 👋🏼 

>  _I am specifically trying to hunt down details regarding fill, fillForUpdate methods on a nova resource_
